src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require('http-proxy-middleware')
```

Comment:
I'm not privy to these proxies, but is this file used anywhere in the codebase? or is it picked automatically by react?

Reply:
The setUpProxy.js file is a development configuration file used by the development server provided by create-react-app (CRA). I installed the package because I was getting CORS error on my side when trying to access the hosted API.
In production, the setUpProxy.js file is not used or read.
The file isn't imported anywhere in the codebase. Reacts picks it up automatically

Reply:
Awesome, Its good to know that its not used in production.